src/pieces/commandManager.ts

```diff
@@ -14,6 +14,18 @@ async function register(bot: Client): Promise<void> {
 		bot.emit('error', error);
 	}
 
+	bot.on('messageCreate', async msg => {
+		if (msg.content.substring(0, 8).toLowerCase().includes('s;enroll')) {
+			// eslint-disable-next-line max-len
+			msg.reply(`If you're trying to run a Sage command, we've moved over to using slash commands. If you're trying to enroll in a course, please use the dropdowns in <#${CHANNELS.ROLE_SELECT}> instead!`)
```

Comment:
You can avoid the eslint issue by hitting enter mid-string, the literal will continue across both lines, though there might be some formatting kinks (those can easily be fixed though)

Reply:
I did, but it had the newline in the discord message so I just disabled eslint. Unsure on how to solve that; suggestions welcome

Reply:
concat the string poterntially?